setup.cfg

```diff
@@ -30,11 +32,12 @@ packages =
 
 [options.entry_points]
 console_scripts =
+    validate-schema = fms_yaml_tools.schema.validate_schema:valyaml
```

Comment:
I think this program should be replace what is currently in `is-valid-diag-table-yaml`

Reply:
It can replace `is-valid-diag-table-yaml`.  I think if both are working for now, we don't need to remove the old one.  If we update the diag table, then we can remove `is-valid-diag-table-yaml`